python_on_whales/command_line_entrypoint.py

```diff
@@ -1,6 +1,11 @@
 from typing import Optional
 
-import typer
+try:
+    import typer
+except ImportError:
+    raise ImportError(
+        "The `python-on-whales` CLI, requires the 'cli' extra. `pip install python-on-whales[cli]`."
+    )
```

Comment:
There should be a `from` here, either `from None` or `from exc` (making it `except ImportError as exc`).